step-certificates/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: step-certificates
-version: 1.17.2
+version: 1.17.3
 appVersion: 0.17.2
```

This will be 0.17.3, I can update it later.

```suggestion
appVersion: 0.17.3
```